DFC.ServiceTaxonomy.GraphSync/GraphSyncers/Parts/EponymousPartGraphSyncer.cs

```diff
@@ -78,9 +78,20 @@ public EponymousPartGraphSyncer(IContentManager contentManager)
 
                 JToken? fieldContent = ((JProperty)field).FirstOrDefault();
                 JProperty? firstProperty = (JProperty?)fieldContent?.FirstOrDefault();
+
+                string? keyName = ((JProperty)field).Name;
+
                 if (firstProperty == null)
                     continue;
 
+                if (!contentTypePartDefinition.PartDefinition.Fields.Any(z => z.Name.Equals(keyName, StringComparison.OrdinalIgnoreCase)))
+                {
+#pragma warning disable CS8625 // Cannot convert null literal to non-nullable reference type.
+                    mergeNodeCommand.Properties.Add(await _graphSyncHelper!.PropertyName(keyName), null);
```

Comment:
Is this really necessary?
I think properties with a null value aren't actually created on the node in neo.
If there is a good reason, we'll have to make sure the change doesn't go missing during merge, as the epoonymous graph syncer has been refactored in an outstanding pr.

Reply:
Now that the cache is clearing we may be able to simply continue here - I shall give it a go. The problem I had was that not passing the property as null to Neo, it was still appearing, but that could be due to caching.